From 9e21dea757de9466f4ba1e39bbfc08afc1c21d02 Mon Sep 17 00:00:00 2001
From: Octavian Purdila <octavian.purdila@nxp.com>
Date: Wed, 8 Feb 2017 12:34:12 +0200
Subject: [PATCH 1394/5242] MLK-13890 ARM: dts: imx6ul-14x14-evk: remove a few
 pins from hoggrp-1

commit  c550a6e19fddd1a8b9cac7832d7604f3864ebcf9 from
https://source.codeaurora.org/external/imx/linux-imx.git

They belong in the usdhc1 group, where they were already present and
which causes the usdhc1 device to fail probe:

[    1.484978] imx6ul-pinctrl 20e0000.iomuxc: pin MX6UL_PAD_UART1_RTS_B already requested by 20e0000.iomuxc; cannot claim for 2190000.usdhc
[    1.496166] imx6ul-pinctrl 20e0000.iomuxc: pin-36 (2190000.usdhc) status -22
[    1.501959] imx6ul-pinctrl 20e0000.iomuxc: could not request pin 36 (MX6UL_PAD_UART1_RTS_B) from group usdhc1grp  on device 20e0000.iomuxc
[    1.513186] sdhci-esdhc-imx 2190000.usdhc: Error applying setting, reverse things back

Note that out internal commit cf65c8d ("MLK-11300-02 ARM: dts: imx:
add imx6ul board dts support") is similar with the upstream commit
a5fcccb ("ARM: dts: imx: add imx6ul and imx6ul evk board support") but
since it has more nodes defined we had to integrate parts of it
nonetheless.

Signed-off-by: Octavian Purdila <octavian.purdila@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-evk.dtsi |    3 ---
 1 file changed, 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
index dbbe14c..5400438 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
+++ b/arch/arm/boot/dts/imx6ul-14x14-evk.dtsi
@@ -432,9 +432,6 @@
 
 	pinctrl_hog_1: hoggrp-1 {
 		fsl,pins = <
-			MX6UL_PAD_UART1_RTS_B__GPIO1_IO19	0x17059 /* SD1 CD */
-			MX6UL_PAD_GPIO1_IO05__USDHC1_VSELECT	0x17059 /* SD1 VSELECT */
-			MX6UL_PAD_GPIO1_IO09__GPIO1_IO09        0x17059 /* SD1 RESET */
 			MX6UL_PAD_SNVS_TAMPER0__GPIO5_IO00	0x17059
 		>;
 	};
-- 
1.7.9.5

